cmd/uhc/cluster/list/list.go

```diff
@@ -195,23 +261,13 @@ func run(cmd *cobra.Command, argv []string) {
 		managed = false
 	}
 
-	// DEFAULT PADDING, CHANGE THIS VALUE TO CLIP MORE OR LESS
-	paddingByColumn := []int{
-		35,
-		40,
-		80,
-		35,
-		15,
-	}
+	// Update our column name displaying variable:
+	args.columns = strings.Replace(args.columns, " ", "", -1)
+	colUpper := strings.ToUpper(args.columns)
+	columnNames := strings.Split(colUpper, ",")
```

Comment:
This means that column names will be `API.URL`, `VERSION.ID`, including the dot. Is that what you want? I'd say it would be better to have a map that contains custom column names for well known columns. Anything that isn't in that map can be then calculated using the conversion to upper case that you are suggesting, but I think that it should be with spaces instead of dots. Anyhow, I won't block you patch because of that. Consider doing it in a separate patch.

Reply:
Actually, whenever we return false.  The column will be set to "NONE".
```golang
for _, element := range iter {
  value, status := findMapValue(jsonBody, element)
  if !status {
    value = "NONE"
  }
  thisCluster = append(thisCluster, value)
  thisCluster = updateRowPad([]int{args.padding}, thisCluster)
}
```